src/core/lib/security/security_connector/security_connector.h

```diff
@@ -244,6 +244,9 @@ tsi_peer tsi_shallow_peer_from_ssl_auth_context(
     const grpc_auth_context* auth_context);
 void tsi_shallow_peer_destruct(tsi_peer* peer);
 
```

Comment:
sorry I was not clear. Please delete line 246 & 247.

ssl_host_matches_name belongs to functions that are testing only.